packages/sku/lib/banner.js

```diff
@@ -2,7 +2,13 @@ const chalk = require('chalk');
 const wrap = require('wrap-ansi');
 const identString = require('indent-string');
 
-module.exports = (type, heading, messages = []) => {
+/**
+ * @param {'info' | 'error' | 'warning'} type
+ * @param {string} heading
+ * @param {string[]} messages
+ *
+ * */
```

Comment:
```suggestion
 */
```

There are a few more instances of this, but I'll stop here.